### PR TITLE
Adds a 'safe' ash storm variant.

### PR DESCRIPTION
**Pull request by Cameron653**
For admemes.

### CODE REVIEW

File: code/__defines/planets.dm [hunk 1]
```diff
@@ -11,6 +11,7 @@
 #define WEATHER_BLOOD_MOON			"blood moon" // For admin fun or cult later on.
 #define WEATHER_EMBERFALL			"emberfall" // More adminbuse, from TG. Harmless.
 #define WEATHER_ASH_STORM			"ash storm" // Ripped from TG, like the above. Less harmless.
+#define WEATHER_ASH_STORM_SAFE		"ash storm (Safe)" //Safe version of the ash storm. Dimmer.
```
Comment (Heroman3003) [inline code/__defines/planets.dm:14]: considering that name is used ingame for IC stuff, I'd suggest a usage of name like "light ash storm" or something among those lines

File: code/modules/planet/virgo3b_vr.dm [hunk 2]
```diff
@@ -479,6 +480,27 @@ var/datum/planet/virgo3b/planet_virgo3b = null
 
 			L.inflict_heat_damage(rand(1, 3))
 
+/datum/weather/virgo3b/ash_storm_safe
+	name = "ash storm (Safe)"
```
Comment (Heroman3003) [inline code/modules/planet/virgo3b_vr.dm:484]: ditto

File: code/modules/planet/virgo3c_vr.dm [hunk 3]
```diff
@@ -475,6 +476,30 @@ var/datum/planet/virgo3c/planet_virgo3c = null
 			L.inflict_heat_damage(1)
 			to_chat(L, "<span class='warning'>Smoldering ash singes you!</span>")
 
+
+
+//A non-lethal variant of the ash_storm. Stays on indefinitely.
+/datum/weather/virgo3c/ash_storm_safe
+	name = "ash storm (Safe)"
```
Comment (Heroman3003) [inline code/modules/planet/virgo3c_vr.dm:483]: threetto

File: code/modules/planet/virgo4_vr.dm [hunk 4]
```diff
@@ -452,6 +453,28 @@ var/datum/planet/virgo4/planet_virgo4 = null
 
 			L.inflict_heat_damage(rand(1, 3))
 
+//A non-lethal variant of the ash_storm. Stays on indefinitely.
+/datum/weather/virgo4/ash_storm_safe
+	name = "ash storm (Safe)"
```
Comment (Heroman3003) [inline code/modules/planet/virgo4_vr.dm:458]: quarto

File: code/__defines/planets.dm (hunk 1 above)
Comment (Cameron653) [reply code/__defines/planets.dm:14]: Ah, makes sense. I'll go ahead and change that real quick

**Comment by Cameron653** [comment]
Name changed to "light ash storm"

**Pull request merged**